lang/lang_test.go

```diff
@@ -23,6 +24,29 @@ func TestAddTranslations(t *testing.T) {
 	assert.Equal(t, "Match2", L("Test2"))
 }
 
+func TestLocalize_Default(t *testing.T) {
```

Comment:
Strangely, this test passes even with the code change commented out, and I played around with various permutations of the test code here and I couldn't get it to fail for the old code and pass for the new code, so I'm not sure what it's actually testing

Reply:
I was attempting to test the fallback logic separately from the init code - instead I have just pulled in a translation from Weblate and checked the fallback based on internal translations.